cmd/build/helmify/replacements.go

```diff
@@ -57,6 +57,8 @@ var replacements = map[string]string{
 
 	"- HELMSUBST_MUTATION_ENABLED_ARG": `{{ if not .Values.disableMutation}}- --operation=mutation-webhook{{- end }}`,
 
+	"- HELMSUBST_METRICS_BACKEND_ARG": `{{- range $backend := .Values.metricsBackends -}}{{ printf "%s%s" "- --metrics-backend=" $backend | nindent 8 }}{{- end }}`,
```

Comment:
nit: should we try to be consistent like how other variables are added? https://github.com/open-policy-agent/gatekeeper/blob/b5ab5608421d415f34008e399a02b4a57a9e511f/cmd/build/helmify/replacements.go#L154-L157

Reply:
good idea, working on it